src/main/java/io/debezium/connector/vitess/transaction/Gtid.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Debezium Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.debezium.connector.vitess.transaction;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+class Gtid {
+
+    public String getVersion() {
+        return version;
+    }
+
+    private String version = "";
+
+    public Set<String> getHosts() {
+        return hosts;
+    }
+
+    private Set<String> hosts = new HashSet();
+
+    public List<String> getSequenceValues() {
```

Comment:
It would be good to give an example of `transactionId` and the expected value of `version`, `sequenceValues`